packages/elements-core/src/components/TryIt/Parameters/parameter-utils.ts

```diff
@@ -116,3 +119,48 @@ export function mapSchemaPropertiesToParameters(
     ...(required?.includes(name) && { required: true }),
   }));
 }
+
+export function toParameterSpec(jsonTreeNode: RegularNode): ParameterSpec {
+  const isBoolean = jsonTreeNode.primaryType === 'boolean';
+  // TODO: Why does ParameterSpec assume JSONSchema7 and ignore 6 and 4?
+  const schema = !isBoolean ? (jsonTreeNode.fragment as JSONSchema7) : undefined;
```

Comment:
I don't like this cast, but this doesn't seem like the right time to try and remove the JSON Schema 7 assumption.